.github/workflows/test.yaml

```diff
@@ -37,11 +37,11 @@ jobs:
             dependencies: highest
```

Comment:
this is not the line for this comment but I cant comment on the line I want:

You should remove '7.2' from php-versions array, and you should bump the lowest build to php 7.3.